src/Api/SecretsManager/Models/Request/SecretUpdateRequestModel.cs

```diff
@@ -20,7 +20,7 @@ public class SecretUpdateRequestModel
 
     public Guid[] ProjectIds { get; set; }
 
-    public Secret ToSecret(Guid id)
+    public Secret ToSecret(Guid id, Guid organizationId)
```

Comment:
With the updates to the `UpdateSecretCommand` that I've suggested, we wouldn't need this request to model to change.